components/hab/src/cli.rs

```diff
@@ -237,6 +237,16 @@ pub fn get() -> App<'static, 'static> {
             (about: "Commands relating to Habitat origin keys")
             (aliases: &["o", "or", "ori", "orig", "origi"])
             (@setting ArgRequiredElseHelp)
+            (@subcommand delete =>
+                (about: "Removes an unused/empty origin")
+                (aliases: &["del", "dele"])
+                (@arg ORIGIN: +required {valid_origin} "The origin name")
+                (@arg BLDR_URL: -u --url +takes_value {valid_url}
```

Comment:
CLAP groks default values, and then you don't have to include them in the string yourself
```suggestion
                (@arg BLDR_URL: -u --url +takes_value {valid_url} default_value[https://bldr.habitat.sh]
```
We don't use this elsewhere, but we really should. I don't think there's a downside to gradually changing it, but you can take this or leave it.